.github/workflows/release.yaml

```diff
@@ -3,59 +3,11 @@ name: Release
 on:
   push:
     tags:
-      - '*'
+      - "*"
 
 jobs:
-  goreleaser:
-    runs-on: macos-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-      - name: Set up Go
-        uses: actions/setup-go@v5
-        with:
-          go-version: 1.22.x
-      - name: Set up Gon
-        run: brew tap conductorone/gon && brew install conductorone/gon/gon
-      - name: Import Keychain Certs
-        uses: apple-actions/import-codesign-certs@v1
-        with:
-          p12-file-base64: ${{ secrets.APPLE_SIGNING_KEY_P12 }}
-          p12-password: ${{ secrets.APPLE_SIGNING_KEY_P12_PASSWORD }}
-      - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v6
-        with:
-          version: "~> v2"
-          args: release --clean
-        env:
-          GITHUB_TOKEN: ${{ secrets.RELENG_GITHUB_TOKEN }}
-          AC_PASSWORD: ${{ secrets.AC_PASSWORD }}
-          AC_PROVIDER: ${{ secrets.AC_PROVIDER }}
-  goreleaser-docker:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-      - name: Set up Go
-        uses: actions/setup-go@v5
-        with:
-          go-version: 1.22.x
-      - name: Docker Login
-        uses: docker/login-action@v1
-        with:
-          registry: ghcr.io
-          username: ${{ github.repository_owner }}
-          password: ${{ secrets.RELENG_GITHUB_TOKEN }}
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
-      - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v6
-        with:
-          version: "~> v2"
-          args: release --clean -f .goreleaser.docker.yaml
-        env:
-          GITHUB_TOKEN: ${{ secrets.RELENG_GITHUB_TOKEN }}
\ No newline at end of file
+  release:
+    uses: ConductorOne/github-workflows/.github/workflows/release.yaml@v2
+    with:
+      tag: ${{ github.ref_name }}
+      lambda: false
```

Comment:
this should only be lambda: false if we don't want to run it as a lambda, but I see you added the Dockerfile.lambda, it should be one or the other

also all the secrets are missing

https://github.com/ConductorOne/baton-azure-devops/blob/main/.github/workflows/release.yaml